### DISCUSSION TITLE
Option to review flashcards anytime

### DISCUSSION THREAD

**Discussion by KaelSE** [st3v3nmw/obsidian-spaced-repetition#196] [Q&A]
Hi,
Although this plugin is actually called 'spaced-repetition', I personally do not often use that feature but just want to review at anytime. It would be great if there is a button somewhere or a toggle in settings to view all cards in a deck regardless of the algorithm. Thanks!

**Comment by mjsr-sr** [comment]
I don't know if this is related but do you know if is there a way to review again without having to manually remove the revision stamps?
For me that is really painful, so if review anytime refers to ignore the stamps or basically recreate them at will I'm in total favor of this feature.

**Reply by safrilth** [reply]
I know some work arround.. i use regex in vscode to automate removing revision stamp for entire vault, that have 2500 flashcard.. its just take about 6 seconds to complete..

If someone not familiar with those step, you can follow this step :
1. download vscode
2. open your vault folder using vscode
3. exclude your .obsidian folder from search/replace, for detailed explanation [here](https://stackoverflow.com/questions/29971600/choose-folders-to-be-ignored-during-search-in-vs-code)
4. Ctrl + Shift + H and enter this regex syntax, for detailed explanation [here](https://stackoverflow.com/questions/1084741/regexp-to-strip-html-comments)

Note : because this is a destuctive method (replace), so don't forget to backup your vault first..
Hope this will help someone who read this discussion.. 😉

**Comment by st3v3nmw** [comment]
I've opened an issue on this [here](https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/279), this should be implemented pretty soon after I fix [this](https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/129).

**Comment by FUNageddon** [comment]
Will this become a new feature anytime soon? This would be very very useful.

**Comment by brando-ap** [comment]
Easy way to remove all the `<!--SR:!2023-03-08,2,164-->` is by using vim.
In console, use `:%g/<.*>/d`.
This will delete the text and lines, resetting the times.
